tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -484,6 +484,61 @@ public function data_provider_not_supported_webp_images() {
 		yield 'GIFT image' => array( TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/earth.gif' );
 	}
 
+	/**
+	 * Ensure jpg image is replaced if the request is for the frontend context.
+	 *
+	 * @group webp_uploads_update_image_references
+	 *
+	 * @test
+	 */
+	public function it_should_not_replace_image_references_when_request_is_for_frontend_context() {
+		$this->mock_action_template_redirect();
```

Comment:
See my other comment on the missing `wp_head` check above, you should also add a `mock_action_wp_head()` helper method and call that here too.